src/js/actions/layers.js

```diff
@@ -519,7 +521,7 @@ define(function (require, exports) {
             } else if (layer.kind === layer.layerKinds.TEXT) {
                 property = "boundingBox";
             } else {
-                property = "boundsNoEffects";
+                property = "boundsNoMask";
```

Comment:
if we're no longer using boundsNoEffects, can you remove it from the properties list above?